Makefile

```diff
@@ -49,18 +52,34 @@ build: docker_prepare
 # Docker
 docker_prepare:
 	-mkdir -p $(BUILD_DIR)/deb
-	docker-compose up --force-recreate -d
+	docker-compose $(DOCKER_COMPOSE_CONFIG) up --force-recreate -d
 ifeq ($(DEBUG),1)
 	-docker inspect $(CONTAINER_NAME)
 	-$(DOCKER_EXEC) "cat ~/.netrc"
 endif
 
 docker_image:
-	docker build -t $(IMAGE_NAME) ./docker
+	docker build -t $(PROJECT_NAME) ./docker
+
+docker_image_windows:
+	docker build -t $(PROJECT_NAME).windows -f docker/Dockerfile.windows .
 
 # GitHub
 github_upload:
-	$(DOCKER_EXEC) ". /usr/local/bin/tools.sh && create_release_and_upload_asset $(TRAVIS_TAG) $(DEB_NAME) $(DEB_MIME)"
+ifneq ($(BUILD_TAG),)
+ifeq ($(TRAVIS_OS_NAME),linux)
+	$(DOCKER_EXEC) "source docker/tools.sh && \
+	PROJECT_NAME=$(PROJECT_NAME) TRAVIS_OS_NAME=$(TRAVIS_OS_NAME) \
+	create_github_release $(BUILD_TAG)"
```

Comment:
This needs to go in the `scripts/ci/linux/after_success.sh` after the other upload code is done. Not in another makefile started differently

Reply:
Sure, if we want to stick to the original scripts and have it all in one place.
I've intentionally kept them as they were (the scripts), thus the makefile approach. Additionally the makefiles make it easier to run just parts of the different stages/events of the build (helps with testing etc.). Dropping it also.